### DISCUSSION TITLE
iOS App is crashing on launch if build is generated from fastlane but works fine if build is generated from xcode

### DISCUSSION THREAD

**Discussion by ahmedshoeib** [fastlane/fastlane#18866] [Q&A]
Dears,

I'm got below crash from device logs as my app is crashing on launch if build is generated from fastlane but works fine if build is generated from xcode

Can you please support what's the issue and why it's only crashing  if build is generated from fastlane ?

Crash log :

> 
Date/Time:           2021-06-08 10:56:04.6757 +0400
Launch Time:         2021-06-08 10:56:04.4340 +0400
OS Version:          iPhone OS 14.4.2 (18D70)
Release Type:        User
Baseband Version:    5.01.01
Report Version:      104

Exception Type:  EXC_BAD_ACCESS (SIGSEGV)
Exception Subtype: KERN_INVALID_ADDRESS at 0x0000000106540440
Termination Signal: Segmentation fault: 11
Termination Reason: Namespace SIGNAL, Code 0xb
Terminating Process: exc handler [1767]
Highlighted by Thread:  0

Backtrace not available

Unknown thread crashed with ARM Thread State (64-bit):
    x0: 0x0000000106540445   x1: 0x0000000106540440   x2: 0x00000000000006d8   x3: 0x000000016ce32128
    x4: 0x0000000000000073   x5: 0x0000000000000007   x6: 0x0000000000000000   x7: 0x0000000000000000
    x8: 0x0000000000000800   x9: 0x72662e746e65696c  x10: 0x2f6b726f77656d61  x11: 0x6f42746168435342
   x12: 0x72662e746e65696c  x13: 0x2f6b726f77656d61  x14: 0x6f42746168435342  x15: 0x00746e65696c4374
   x16: 0x0000000000000004  x17: 0x0000000000000000  x18: 0x0000000000000000  x19: 0x0000000000000128
   x20: 0x000000016ce32168  x21: 0x0000000106540445  x22: 0x00000000000006d8  x23: 0x0000000106460000
   x24: 0x0000000000000001  x25: 0x000000016ce32908  x26: 0x0000000106460778  x27: 0x0000000106460000
   x28: 0x0000000000000000   fp: 0x000000016ce31c20   lr: 0x0000000106438e34
    sp: 0x000000016ce31c00   pc: 0x00000001064394e4 cpsr: 0x80000000
   esr: 0x00000000  Address size fault

Binary images description not available

Error Formulating Crash Report:
Failed to create CSSymbolicatorRef - corpse still valid ¯\_(ツ)_/¯

EOF

**Comment by crazymanish** [comment]
Hi @ahmedshoeib 

I am requesting you to try a sample app using Fastlane and check if that sample app is crashing on launch or not.
- If sample app is NOT crashing on launch - then, there is some issue with your project-settings
- If sample app is crashing on launch - then, we have a much bigger issue ()

Waiting for your reply! 
Thank you

**Reply by ahmedshoeib** [reply]
same fastlane code was working fines for me after some updates and adding some pods got this issue and didn't get what's the issue and why it's crashing

**Reply by crazymanish** [reply]
> some updates and adding some pods got this issue 

Hi, in that case, I request you to install those pods in sample app and try if sample app is crashing on launch or not.

**Comment by Shyam-Khokhariya** [comment]
Same issue for me. I updated my react native version and now it is crashing the app sometimes. Here is what error message i see on crash:
Exception -
EXC_BAD_ACCESS: Exception 1, Code 1, Subcode 140737488398928 >
KERN_INVALID_ADDRESS at 0x80000000aa50.
